web/styles/popovers.css

```diff
@@ -1190,4 +1218,120 @@ ul {
         width: 16px;
         height: 16px;
     }
+
+    .zulip-icon {
+        position: relative;
+        top: -1px;
+    }
+}
+
+#gear-menu-dropdown {
+    padding: 5px 0;
+    box-shadow:
+        0 2.7798px 4.1129px 0 hsl(0deg 0% 0% / 10%),
+        0 5.5113px 8.5783px 0 hsl(0deg 0% 0% / 9%),
+        0 8.4377px 13.9271px 0 hsl(0deg 0% 0% / 11%),
+        0 12.177px 21.4737px 0 hsl(0deg 0% 0% / 11%),
+        0 18.7257px 35.4802px 0 hsl(0deg 0% 0% / 15%),
+        0 41px 80px 0 hsl(0deg 0% 0% / 20%);
+
+    .org-info {
+        padding-top: 4px;
+
+        & li {
+            display: flex;
+            justify-content: center;
+            font-size: 15px;
+            font-style: normal;
+            font-weight: 400;
+            line-height: 16px; /* 114.286% */
+            letter-spacing: 0.28px;
+
+            &:focus-within {
+                background: var(--color-background-hover-dropdown-menu);
+            }
+
+            & a {
+                padding: 2px 0;
+                flex-grow: 1;
+                text-align: center;
+
+                &:hover {
+                    text-decoration: none;
+                    background: var(--color-background-hover-dropdown-menu);
+                    outline: none;
+                }
+
+                &:focus {
```

Comment:
I think we should be using `focus-visible` in almost all cases; that is, we let the browser decide whether to render based on the input type (generally, that means a mouse interaction won't show the focus style, which is almost certainly what we want).

Reply:
Thanks for the suggestion, Aman! I'll think about it and amend #27117 when I've got something actionable that I've at least poked at to make sure I don't lead anyone astray.